packages/flutter/lib/src/material/divider.dart

```diff
@@ -12,11 +13,11 @@ import 'theme.dart';
 /// In the material design language, this represents a divider.
 ///
 /// Dividers can be used in lists, [Drawer]s, and elsewhere to separate content
-/// vertically. To create a one-pixel divider between items in a list, consider
-/// using [ListTile.divideTiles], which is optimized for this case.
+/// vertically or horizontally. To create a one-pixel divider between items in
+/// a list, consider using [ListTile.divideTiles], which is optimized for this case.
 ///
-/// The box's total height is controlled by [height]. The appropriate padding is
-/// automatically computed from the height.
+/// The box's total width or height is controlled by [height]. The appropriate
```

Comment:
Maybe "The divider's total length is controlled by [height]."  It's a shame we can't rename `height` to `length` without breaking though.

Reply:
I know...